daemon/server/router/checkpoint/checkpoint_routes.go

```diff
@@ -39,6 +39,9 @@ func (cr *checkpointRouter) getContainerCheckpoints(ctx context.Context, w http.
 	if err != nil {
 		return err
 	}
+	if checkpoints == nil {
```

Comment:
Probably both OK, but I usually pick `if len(xxx) == 0` for these; ISTR there was a corner case where it was not `nil` but a nil slice, and `len()` detected both, but `== nil` didn't